src/poker-evaluator.ts

```diff
@@ -64,11 +66,61 @@ function evaluate(cardValues: number[]): EvaluatedHand {
   if (cardValues.length === 5 || cardValues.length === 6) {
     p = evalCard(p);
   }
+  const handType = (p === ROYAL_FLUSH_SCORE) ? 10 : p >> 12;
 
   return {
-    handType: p >> 12,
+    handType: handType,
```

Comment:
Thanks for calling this out in [your comment](https://github.com/Sukhmai/poker-evaluator/pull/2#issuecomment-1828755662). I definitely agree let's revert this change because we don't want to create breaking changes for our users. This also broke two of the unit tests that expected a "straight flush" hand. Run `npm test` to see. You can handle the royal flush case in your application by looking for `handType == 9` and `handRank == 10`.